releasenotes/notes/remove-pulse-qpy-07a96673c8f10e38.yaml

```
@@ -0,0 +1,8 @@
+---
+upgrade_qpy:
+  - |
+    With the removal of Pulse in Qiskit 2.0, support for serializing ``ScheduleBlock`` programs
+    via the :func:`qiskit.qpy.dump` function has been removed. Users can still load payloads
+    containing pulse gates using the :func:`qiskit.qpy.load` function, however those will be
+    treated as undefined custom instructions. Loading ``ScheduleBlock`` payloads is not supported
```

The term we normally use for this is opaque instructions. For an instruction that contains no definition or other details.

```suggestion
    treated as opaque custom instructions. Loading ``ScheduleBlock`` payloads is not supported
```